src/main.rs

```diff
@@ -124,15 +126,8 @@ fn command_build(root: String) -> Result<(), Error> {
     let compiled = crate::project::compile(srcs)?;
 
     // Delete the gen directory before generating the newly compiled files
-    let gen_dir = root_path
-        .parent()
-        .ok_or_else(|| Error::FileIO {
-            action: error::FileIOAction::FindParent,
-            kind: error::FileKind::Directory,
-            path: root_path.clone(),
-            err: None,
-        })?
-        .join("gen");
+    let gen_dir = root_path.join("gen");
```

Comment:
This was actually a bug I found.  It was always a noop because it was looking outside the project dir

Reply:
Nice catch!